automobile/openstreetmap-importer.md

```diff
@@ -23,19 +23,22 @@ You can use several arguments with this script:
+| --outputFile        | Specifies the name of the generated world file                                                                                       | If not specified, the generated world is called "map.wbt"                 |
+| --configFile        | Specifies which configuration file to use                                                                                            | If not specified, tries to use the configuration file called "config.ini" |
+| --splineSubdivision | Defines the spline subdivision used for roads, rivers, etc.                                                                          | A default value of 4 is used                                              |
+| --layerHeight       | Defines the height of a layer (the 'layer' tag is ignored if set to 0)                                                               | A default value of 5.0 is used                                            |
+| --noForest          | Does not include the forests in the generated world                                                                                  | By default, forests are included                                          |
+| --noRoad            | Does not include the roads in the generated world                                                                                    | By default, roads are included                                            |
+| --noArea            | Does not include the areas (water area, landuse, etc.) in the generated world                                                        | By default, areas are included                                            |
+| --noTree            | Does not include the isolated trees in the generated world                                                                           | By default, isolated trees are included                                   |
+| --noBarrier         | Does not include the barriers (fence, wall, etc.) in the generated world                                                             | By default, barriers are included                                         |
+| --noRiver           | Does not include the rivers in the generated world                                                                                   | By default, rivers are included                                           |
+| --noBuilding        | Does not include the buildings in the generated world                                                                                | By default, buildings are included                                        |
+| --noRoadIntersection| Does not generate complex road intersections                                                                                         | By default, complex road intersections are generate                       |
```

Comment:
The comment appeared on my screen at the exact same moment I merged, I will fix this in a new PR.